src/components/AnnotationViewer.vue

```diff
@@ -292,6 +311,82 @@ export default class AnnotationViewer extends Vue {
       this.drawNewConnections(displayedIds);
     }
     this.annotationLayer.draw();
+    this.drawTooltips();
+  }
+
+  drawTooltips() {
+    const oldFeatures = this.textLayer.features();
+
+    if (this.shouldDrawTooltips) {
+      // Avoid checking the store for each annotation
+      const all = this.tooltipOnAll;
```

Comment:
I don't really understand the necessity of these variables, can you explain?

Reply:
They are not necessary, but because we have to use them in the loop over all annotations, I think it is more efficient to fetch them from the store once and then use them as local variables.
The performance impact is not measured though.